scripts/zones/Abyssea-Grauberg/IDs.lua

```diff
@@ -102,30 +102,28 @@ zones[xi.zone.ABYSSEA_GRAUBERG] =
-            --  [17818085] = {  'qm5',             { 3265 },                                                              { }, 17818045 }, -- Teekesselchen
-            --  [17818086] = {  'qm6',             { 3267 },                                                              { }, 17818046 }, -- Minaruja
-            --  [17818087] = {  'qm7',             { 3269 },                                                              { }, 17818047 }, -- Xibalba
-            --  [17818088] = {  'qm8',             { 3270 },                                                              { }, 17818048 }, -- Ika-Roa
-            --  [17818089] = {  'qm9',             { 3271 },                                                              { }, 17818049 }, -- Lorelei
-            --  [17818090] = { 'qm10',             { 3273 },                                                              { }, 17818050 }, -- Burstrox Powderpate
-            --  [17818091] = { 'qm11',                  { }, { xi.ki.WARPED_CHARIOT_PLATE, xi.ki.SHATTERED_IRON_GIANT_CHAIN }, 17818051 }, -- Raja
-            --  [17818092] = { 'qm12',                  { },                                    { xi.ki.VENOMOUS_HYDRA_FANG }, 17818052 }, -- Alfard
-            --  [17818093] = { 'qm13',                  { },                                      { xi.ki.VACANT_BUGARD_EYE }, 17818053 }, -- Azdaja
-            --  [17818094] = { 'qm14',                  { },                              { xi.ki.VARIEGATED_URAGNITE_SHELL }, 17818054 }, -- Amphitrite
-            --  [17818095] = { 'qm15',                  { }, { xi.ki.WARPED_CHARIOT_PLATE, xi.ki.SHATTERED_IRON_GIANT_CHAIN }, 17818055 }, -- Raja
-            --  [17818096] = { 'qm16',                  { },                                    { xi.ki.VENOMOUS_HYDRA_FANG }, 17818056 }, -- Alfard
-            --  [17818097] = { 'qm17',                  { },                                      { xi.ki.VACANT_BUGARD_EYE }, 17818057 }, -- Azdaja
-            --  [17818098] = { 'qm18',                  { },                              { xi.ki.VARIEGATED_URAGNITE_SHELL }, 17818058 }, -- Amphitrite
-            --  [17818099] = { 'qm19',                  { }, { xi.ki.WARPED_CHARIOT_PLATE, xi.ki.SHATTERED_IRON_GIANT_CHAIN }, 17818059 }, -- Raja
-            --  [17818100] = { 'qm20',                  { },                                    { xi.ki.VENOMOUS_HYDRA_FANG }, 17818060 }, -- Alfard
-            --  [17818101] = { 'qm21',                  { },                                      { xi.ki.VACANT_BUGARD_EYE }, 17818061 }, -- Azdaja
-            --  [17818102] = { 'qm22',                  { },                              { xi.ki.VARIEGATED_URAGNITE_SHELL }, 17818062 }, -- Amphitrite
+                [17818081] = { 'qm_ironclad_sunderer',                            { xi.item.TEEKESSELCHEN_FRAGMENT, xi.item.DARKFLAME_ARM },                                                              { }, 17818041 }, -- Ironclad Sunderer
```

Comment:
There is no reason to uncomment this.  It is unused in the current abyssea global

Reply:
To be clear, the changes to the table are fine. Replacing raw ids with enums and the proper npc names.

However, since the table is unused, it should remain commented out.

Reply:
understood. and thank you for the feedback.

Reply:
Many thanks though for doing that work of changing those magic numbers to their enums (even if the table isn't even used!)

DEATH TO MAGIC NUMBERS! LONG LIVE ENUMS!!!